src/components/Radio.jsx

```diff
@@ -1,3 +1,5 @@
+import { RadioGroup, RadioGroupItem } from "./ui/radio";
+
 const Radio = ({
```

Comment:
this file as a whole shouldn't be needed and should be removed, it creates an extra middle man for the radio component, all logic should be handled by the parent, while the child only handles the rendering